packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/.gitignore

```diff
@@ -0,0 +1,2 @@
+upgrade-walletFactory-permit.json
```

Comment:
question: should we have a global gitignore on `-permit.json`? if so, should we name them `.permit.json` to hint that it's a kind of thing?

Reply:
It _is_ a kind of thing. (There's a static type and could be a pattern). So I'd support renaming all of them together in a separate PR.

Reply:
https://github.com/Agoric/agoric-sdk/pull/8249, draft until this one lands